tests/nightly/security-internal-apparmor/task.yaml

```diff
@@ -0,0 +1,41 @@
+summary: Check internal vendored apparmor in snapd snap
+
+manual: true
```

Comment:
actually I think you can remove this, as soon as this PR is merged, the snapd snap will start being built with apparmor in it, and thus this spread test should start passing in the nightly test runs, @sergiocazzolato do you agree?

Reply:
ping @sergiocazzolato ^

Reply:
I suppose it is fine to leave this as manual and open a followup PR immediately after making it not manual. Also fwiw to respond to this comment you need to not have any pending comments as part of a "review" and then you need to find the original comment/review that I left this on and respond there for some reason, I agree it is silly and unhelpful UI, so I will just resolve this comment for the sake of simplicity